todoman/model.py

```diff
@@ -585,8 +586,11 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             q = ', '.join(['?'] * len(lists))
             extra_where.append('AND files.list_name IN ({})'.format(q))
             params.extend(lists)
-        if urgent:
-            extra_where.append('AND priority = 9')
+        if priority:
+            if priority == 'medium' or priority == '!low':
```

Comment:
You handle `!low` here, but it's impossible that this value is passed because it's invalid according to the validation code you have in `cli.py`.